README.md

```diff
@@ -80,6 +80,12 @@ See the [OpenTelemetry registry](https://opentelemetry.io/registry/?s=net) and
 repo](https://github.com/open-telemetry/opentelemetry-dotnet-contrib) for more
 components.
 
+## Troubleshooting
```

Comment:
promoting to the main readme as well, as this has been hard to find based on several feedback.